python-backend/app/__init__.py

```diff
@@ -45,15 +46,18 @@ def register_extensions(app):
     db.init_app(app)
     jwt.init_app(app)
     apm.init_app(app) if app.config['ELASTIC_ENABLED'] == '1' else None
+    sched.init_app(app)
 
     CORS(app)
     Compress(app)
 
-    if app.config.get('ENVIRONMENT_NAME') == 'prod':
-        sched.init_app(app)
+    if app.config.get('ENVIRONMENT_NAME') == 'test' or app.config.get('ENVIRONMENT_NAME') == 'prod':
```

Comment:
You can probably do something like `condition in ['test', 'prod']` for this.

Reply:
Changed.